.checkpatch.conf

```diff
@@ -18,3 +18,4 @@
 --ignore CONST_STRUCT
 --ignore FILE_PATH_CHANGES
 --exclude ext
+--exclude samples/subsys/canbus/canopen/objdict
```

Comment:
IMHO file generated by external tool and and not follows coding style belongs to `ext`, there must not be other exception. Also, there are alternatives to this, though not so easy, one is to patch the libedssharp tool and the other to write your own script to generate EDS.

Reply:
The generated files will always need to contain typedefs and use the types from CANopenNode, no matter what script generated them.

I can move them to ext, though, if that is preferable.